### ISSUE TITLE
Rylai's Wheel bug

### ISSUE THREAD

**Issue by cyrilvoynov** [ValveSoftware/Dota-2#1793]
I just spin the Rylai's wheel and it stopped on the Arcane. But instead of getting it, all buttons except the "back to the main menu" button just disappeared. Will I get my Arcane thing? And what happened here? Rylai's wheel disappeared also.  

https://www.youtube.com/watch?v=LXiCLEkSuxY&feature=youtu.be

**Comment by glubsy** [comment]
No, it's most likely a network error. You won't get the arcana because the game decides as soon as you click "spin the wheel" what you will get at the end, the animation has nothing to do with any of it. In this case nothing had been decided by the game.
